### DISCUSSION TITLE
Join the Fun: Let’s Play Universe Bingo and Watch Together!

### DISCUSSION THREAD

**Discussion by samus-aran** [community/community#74691] [Announcements]
# [Play Universe Bingo Online here](https://universebingo.dev/) 🎲 

The ultimate developer event of the year, [GitHub Universe](http://githubuniverse.com) is live, and we have a brilliant idea to level up the excitement. How about we combine the fun of playing along with the buzz of watching the **[GitHub Universe Live](https://reg.githubuniverse.com/flow/github/universe23/sessioncatalog/page/watch?utm_source=Discussion&utm_medium=GitHub&utm_campaign=livestream)**?

![Universe_bingo](https://github.com/community/community/assets/4041275/327c55a6-abd7-4db2-9ac4-9edea1278779)
> Alt Text: The image displays a bingo board that include squares of GitHub Universe actions on a black background

## 🤩 Why You Should Join:

* **Double the fun:** Witness groundbreaking announcements while having a little fun.
* **Community spirit:** Team up in the comments and share what quotes you hear to win.
* **Interactive discussion:** Share your thoughts on the keynote as it unfolds.

## 🎯 How to Join:

Open up [GitHub Universe Bingo](https://universebingo.dev/) along side the [livestream](https://reg.githubuniverse.com/flow/github/universe23/sessioncatalog/page/watch?utm_source=Discussion&utm_medium=GitHub&utm_campaign=livestream) and comment here what you hear that fills a Bingo square 💖💬👇

**Comment by marielenna** [comment]
how fun 🥳

**Comment by jricciardi** [comment]
Free space is really just the word "Copilot" 🚀

**Comment by gaurav-ketkar** [comment]
Ha! This is a great idea 😄 🚀 
I've definitely heard "developers" more than once already! 

<img width="415" alt="image" src="https://github.com/community/community/assets/9720857/625faec0-da0a-4112-923c-f33046448a21">

**Comment by jricciardi** [comment]
"Mac Laptop Onstage" - Spotted in the chat with Postman CEO Abhinav Asthana! 


![Screenshot 2023-11-08 at 11 05 47 AM](https://github.com/community/community/assets/31521880/a176d829-c179-4962-93f4-9e10cd4cd08e)

**Comment by ghostinhershell** [comment]
![screenshot of Farhan Thawar's Universe talk on "Empowering devs with AI"](https://github.com/community/community/assets/14021469/74c19d51-0ab9-4712-a137-a582cbf17e5e)

GitHub Copilot Salute! 🖖🏾

**Comment by novium2** [comment]
Closing this as it's almost time for Universe 2024! If you have any questions, consider starting a [new post](https://github.com/orgs/community/discussions/new/choose).
